<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SabirMulla" ignored="0" name="retrymultiple" tests="6" failures="1" timestamp="10 Oct 2022 03:46:45 GMT" time="0.093" errors="0">
  <testcase name="f" time="0.016" classname="retry.NewTest">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.003" classname="retry.NewTest">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.002" classname="retry.NewTest">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.002" classname="retry.NewTest">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.003" classname="retry.NewTest">
    <skipped/>
  </testcase> <!-- f -->
  <testcase name="f" time="0.003" classname="retry.NewTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at retry.NewTest.f(NewTest.java:10)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- f -->
</testsuite> <!-- retrymultiple -->
